### DISCUSSION TITLE
Runway resource

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5479] [Q&A] [answered]
After i created new nav on the sidebar using Runway, it is generated successfully but no blueprints for it and i want to customize some fields for it. How can i do that please ?

**Comment by duncanmcclean** [comment, marked as answer]
For any third-party addon issues, please open an issue on the addon's repository. 

This repository is only for issues/discussions related to Statamic itself.

**Reply by HazemMohamed252** [reply]
OK. Thanks
